storage/src/context/merkle/mod.rs

```diff
@@ -15,7 +15,7 @@ pub mod merkle_storage_stats;
 // Tree must be an ordered structure for consistent hash in hash_tree.
 // The entry names *must* be in lexicographical order, as required by the hashing algorithm.
 // Currently immutable OrdMap is used to allow cloning trees without too much overhead.
-pub type Tree = im::OrdMap<String, Arc<Node>>;
+pub type Tree = im::OrdMap<Rc<String>, Rc<Node>>;
```

Comment:
For now you will need to use `Arc` instead of `Rc` (see: http://ci.tezedge.com/simplestaking/tezedge/3294/1/3), but that will get fixed soon, and then we can go back to `Rc`.